utils/constants.js

```diff
@@ -1,9 +1,14 @@
 const CONSTANTS = {
     PORT: 5000,
     HEALTHCHECK: {
-        Application: ' Magic Mark API',
+        Application: 'Magic Mark API',
         Status: 'Healthy'
-    }
+    },
+    USER: {
```

Comment:
Please, rename into MOCK_USER for code consistence.

Reply:
done